content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -83,33 +83,32 @@ In order to build Atom from source, you need to have a number of other requireme
 ##### Requirements
 
 * macOS 10.9 or later
-* Node.js 6.x or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
-* npm 3.10.x or later (run `npm install -g npm`)
-* Python v2.7.x
+* Node.js 10.12 or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
+* npm 6 or later (run `npm install -g npm`)
+* Python v2.7.x or 3.5+
 * Command Line Tools for [Xcode](https://developer.apple.com/xcode/downloads/) (run `xcode-select --install` to install)
 
 {{/mac}}
 
 {{#windows}}
 
-* Node.js 6.9.4 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
-* Python v2.7.x
+* Node.js 10.12 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
+* Python v2.7.x or 3.5+
   * The `python.exe` must be available at `%SystemDrive%\Python27\python.exe`. If it is installed elsewhere create a symbolic link to the directory containing the `python.exe` using: `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`
 * 7zip (7z.exe available from the command line) - for creating distribution zip files
 * C++ build tools, either:
-  * [Visual C++ Build Tools 2015](https://visualstudio.microsoft.com/vs/older-downloads/)
-  * [Visual Studio 2013 Update 5](https://www.visualstudio.com/downloads/) (Express Edition or better)
-  * [Visual Studio 2015](https://www.visualstudio.com/downloads/) (Community Edition or better)
+  * [Visual C++ Build Tools 2015 or newer](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+  * [Visual Studio 2015 or newer](https://www.visualstudio.com/downloads/) (Community Edition or better)
   * **Unsupported** but more convenient for some who know Node: [windows-build-tools](https://www.npmjs.com/package/windows-build-tools)
```

Comment:
```suggestion
  * [windows-build-tools](https://www.npmjs.com/package/windows-build-tools)
```
This is what we use in the CI builds and we have not had any issues with it. Why we should consider this as unsupported?

https://github.com/atom/atom/blob/master/script/vsts/platforms/windows.yml#L42-L44

Reply:
I agree. This method is so much easier, and as you point out, it's literally the way Atom is tested in CI.

If it's good enough for CI, it should be good enough for end-users. It should be the first option suggested, in my opinion. Would be interested in feedback from the Atom maintainers.

Reply:
That's how the official releases are built! 😄 I think this should even be written as the first option.

Reply:
It's the first recommendation straight from a Microsoft guide document as well: https://github.com/Microsoft/nodejs-guidelines/blob/master/windows-environment.md#environment-setup-and-configuration

The document refers to it as "Microsoft's windows-build-tools":

> Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools] by running `npm install -g windows-build-tools` from an elevated PowerShell (run as Administrator).